### ISSUE TITLE
NVDA fails to navigate to landmarks in EdgeHTML-based content

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#10543]
**Note:** The problem occurs not only in the Microsoft Edge browser, but in all places which display content via EdgeHTML (e.g. the Twitter app for Windows 10). I can provide logs for the issue, if necessary.

### Steps to reproduce:
1. Launch NVDA.
2. Open Microsoft Edge (the EdgeHTML-based one).
3. Load a page with some landmarks in it (nvaccess.org should do it).
4. When the page loads, use the "D" key to navigate between landmarks in the loaded document/page.
5. Invoke the elements list (NVDA+F7) and select the "Landmarks" radio-button.

### Actual behavior:
After step 4, NVDA plays an error and no action is taken.
After step 5, NVDA also plays an error and the list of landmarks fails to load in the elements list dialog.

### Expected behavior:
After step 4, NVDA should go to the next available landmark in the document/page.
After step 5, the list of landmarks should be loaded in the elements list dialog.

### System configuration
- Operating system: Windows 10 Pro version 1909 (build 18363.476), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-19266,67bf1df6, in bulgarian.
- Microsoft Edge version: 44.18362.449.0.
- Microsoft EdgeHTML version: 18.18363.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - v2019.2.1 - it behaves as expected.

**Comment by leonardder** [comment]
Could you please provider a log file?

**Comment by k-kolev1985** [comment]
OK, here is a log attached. Testing done with add-ons disabled and log level set to "Debug".
[NVDA_Log_2019-11-25_#10543.log](https://github.com/nvaccess/nvda/files/3886961/NVDA_Log_2019-11-25_.10543.log)
